python/opentrons_ot3_firmware/constants.py

```diff
@@ -94,3 +96,15 @@ class ErrorCode(int, Enum):
     bad_checksum = 0x02
     invalid_byte_count = 0x03
     invalid_input = 0x04
+
+
+@unique
+class ToolType(int, Enum):
+    """Tool types detected on Head."""
+
+    UNDEFINED_TOOL = (0x00,)
```

Comment:
This is not correct. This creates an enum of tuple values.

Also, notice that  the case is inconsistent with the other enums in this file.